aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIOProperties.java

```diff
@@ -72,6 +72,16 @@ public class S3FileIOProperties implements Serializable {
 
   public static final boolean S3_ACCESS_GRANTS_ENABLED_DEFAULT = false;
 
+  /**
+   * This property is used to enable using the S3 Analytics Accelerator library to accelerate data
+   * access from client applications to Amazon S3.
+   *
+   * <p>For more details, see: https://github.com/awslabs/analytics-accelerator-s3
+   */
+  public static final String S3_ANALYTICS_ACCELERATOR_ENABLED = "s3.analyticsaccelerator.enabled";
```

Comment:
nit: should use - to separate different words, so "s3.analytics-accelerator.enabled"